.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,63 @@
+name: "\U0001F41BBug report"
+description: Create a report to help community improve packages of OptunaHub.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please write a clear and concise description of what the bug is.
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Please write a clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment
+      description: |
+        Please give us your environment information. You can get this information by typing the following.
+        ```
+        python3 -c 'import optuna; print(f"- Optuna version:{optuna.__version__}")'
```

Comment:
It is also better to ask about the version of optunahub (`optunahub.__version__`).